src/parse/string/shift/operand/date.ts

```diff
@@ -1,4 +1,7 @@
+import { isDate } from "node:util/types"
```

Comment:
Can't use node deps from `src`, this would break in prod in a browser 💀 💀 

Should still be fine just using `instanceof Date` though, is there a reason we switched?

Curious if there was a lint error for this, ideally we should have one as it would be easy to miss something like this and the tests would pass